src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -16,6 +16,7 @@ import { debounce } from '@mui/material';
 
 type DIMENSIONS = { width: number; height: number; viewbox: VIEWBOX };
 type VIEWBOX = { x: number; y: number; width: number; height: number };
+export type FLOW = { branchId: string; side: number; p: number };
```

Comment:
1. It's strange to have this class in upper case, I think it's only for constants. Besides we could put any kind of values on arrow one day, so you'd better name it `BranchSideLabel`.
2. You named the value `p`, but it could be any value. And about the side, it would be more compact to have it directly in two parameters, I mean `value1` and `value2`.
3. In typescript is it possible to define value1 and value2 to be either a number or a string?

Reply:
Changed for handling generic branch labels: type renamed to BranchLabel, and now containing 2 generic values, numbers or strings